### PR TITLE
chore(eslint): add eslint and configuration

### LINKED ISSUE #16: Add ESlint into project and add the command for linting and fixing
**Add ESlint into the project with following rules**
- 2 space tab allowed
- line break of only one line allowed in code
- No line breaks at the end of file
- ES6 syntax
- single quotes only
- semicolon always

also, add .eslintignore
which will exclude node_modules and *.json file and any other if needed

If you are working on the issue let me know through the comment will assign the issue to you then

### PR DESCRIPTION
**Pull request by MarineP31**
Closes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # Javascript Basics
 
 This repository was made for beginners to start learning Javascript from Scratch
+
```
Comment (Swap76) [inline README.md]: @MarineP31  Does the linting and fixing problems done by one command only

Comment (MarineP31) [reply README.md]: yes you just have to use the command `npm run lint` in your terminal to see all the eslint issues but I can add `--fix` to the command line to fix automatically the problem

**Comment by MarineP31** [comment]
@Swap76 so I add a new commit with `husky` to create a pre-commit hooks

File: README.md (hunk 1 above)
Comment (Swap76) [reply README.md]: @MarineP31  Go Ahead and add the command for fix also

File: package.json [hunk 2]
```diff
@@ -15,5 +26,10 @@
   "bugs": {
     "url": "https://github.com/Swap76/JavascriptBasics/issues"
   },
-  "homepage": "https://github.com/Swap76/JavascriptBasics#readme"
+  "homepage": "https://github.com/Swap76/JavascriptBasics#readme",
```
Comment (Swap76) [inline package.json]: @MarineP31  Great work with husky :-)

**Comment by Swap76** [comment]
@MarineP31  It was great working with you.

Looking forward to working with you :-)

Star to the repo is highly appreciated.

**Comment by Swap76** [comment]
@MarineP31 The work you have done for the project is highly appreciated. Thanks :-)

Looking forward to working with you on my new project related to Deno https://github.com/Swap76/Learn-Deno/ and together making this better.

Star to the repo is highly appreciated.

**Pull request merged**